stablehlo/reference/tests/add.mlir

```diff
@@ -0,0 +1,270 @@
+// RUN: stablehlo-interpreter-runner -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: add_op_test_f16
+// CHECK-NEXT:  tensor<2x5xf16>
+// CHECK-NEXT:   0.000000e+00 : f16
+// CHECK-NEXT:   -0.000000e+00 : f16
+// CHECK-NEXT:   8.000000e+00 : f16
+// CHECK-NEXT:   8.750000e-01 : f16
+// CHECK-NEXT:   3.999020e-01 : f16
+// CHECK-NEXT:   6.281250e+00 : f16
+// CHECK-NEXT:   0x7C00 : f16
+// CHECK-NEXT:   0x7C00 : f16
+// CHECK-NEXT:   0xFC00 : f16
+// CHECK-NEXT:   0x7C00 : f16
+
+func.func @add_op_test_f16() -> tensor<2x5xf16> {
+  %0 = stablehlo.constant dense<[[0.0, -0.0, 1.0, 0.125, 0.1], [3.141, 0x7C00, 0x7C00, 0xFC00, 0x7C00]]> : tensor<2x5xf16>
+  %1 = stablehlo.constant dense<[[0.0, -0.0, 7.0, 0.75, 0.3], [3.141, 0.0, 0x7C00, 0xFC00, 0x7C00]]> : tensor<2x5xf16>
+  %2 = stablehlo.add %0, %1 : tensor<2x5xf16>
+  func.return %2 : tensor<2x5xf16>
+}
+
+// -----
```

Comment:
Given that the tool handles multiple functions at once, do you need the split markers?

Reply:
Will switch to[ third_party/tensorflow/compiler/mlir/lite/flatbuffer_translate.cc](https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/Tools/mlir-translate/MlirTranslateMain.h). 

Still we support both of split files (comes for free with the MlirTranslate interface) and specifying multiple functions in a module. 

Handling multiple modules is just a convenience. Avoiding it needs us to add constraints like 
  - the module should have only one function, Or 
  - in case of multiple functions, only the first one will be interpreted Or  one with a specific name will be evaluated. 

From user perspective, it means that they do not have to any of the above constraints.